### ISSUE TITLE
Unable to save a file in Grails

### ISSUE THREAD

**Issue by Aszusz** [prominic/Moonshine-IDE#815]
During my Grails development I was suddenly unable to save a file I was working on.
Attaching console log.
[unable-to-save-a-file-log.txt](https://github.com/prominic/Moonshine-IDE/files/6376987/unable-to-save-a-file-log.txt)

**Comment by rat-moonshine** [comment]
Please suggest the OS.

Also, provide steps to reproduce the problem on 'master' branch.

**Comment by piotrzarzycki21** [comment]
> Please suggest the OS.
> 
> Also, provide steps to reproduce the problem on 'master' branch.

It is definitely on Windows.

**Comment by Aszusz** [comment]
Yes, it was on Windows. And I'm sorry, I cannot provide steps. I was learning Grails and half of the time didn't know what I was doing. It was reproducible for that particular set of files (persistent after restart), but after I fixed my code it did not occur again.
